diversity-and-inclusion-policies.md

```diff
@@ -0,0 +1,50 @@
+# Diversity and Inclusion Policy Guidelines
+
+## Intro
+The Confidential Computing Consortium is a community focused on open source
+licensed projects securing DATA IN USE & accelerating the adoption of
+confidential computing through open collaboration.
+
+Every member is welcome; every project meeting our criteria is welcome. 
+We are a transparent, collaborative community.
+
+We as members, contributors, and leaders pledge to make participation
+in our community a harassment-free experience for everyone.
+
+
+## Policies
+The [Project Proposal Template](project-submission-template.md) specifies that
+a project must have a Code of Conduct to be submitted to the Confidential
```

Comment:
Technically speaking this is not strictly true.   Someone could submit and answer "none".  The TAC would then review and decide whether to accept it or not.  The TAC could choose to ask for one before accepting it, but someone can submit a project before then.
```suggestion
a project should have a Code of Conduct to be accepted by the Confidential
```

Reply:
TAC consensus is in support of Dan's proposed wording

Reply:
TAC consensus for "submitted"